.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,8 @@
 			"extensions": [
 				"redhat.vscode-yaml",
 				"tsandall.opa",
-        		"ms-kubernetes-tools.vscode-kubernetes-tools"
+        		"ms-kubernetes-tools.vscode-kubernetes-tools",
+				"esbenp.prettier-vscode"
```

Comment:
Probably still don't want `prettier` in our top level dev container as it's only relevant to the frontend